test/e2e/tools_bucket_web_test.go

```diff
@@ -4,11 +4,24 @@
 package e2e_test
 
 import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
 	"net/http/httptest"
+	"os"
+	"path"
+	"path/filepath"
 	"testing"
+	"time"
 
 	"github.com/cortexproject/cortex/integration/e2e"
 	e2edb "github.com/cortexproject/cortex/integration/e2e/db"
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/pkg/timestamp"
+	v1 "github.com/thanos-io/thanos/pkg/api/blocks"
+	"github.com/thanos-io/thanos/pkg/objstore"
```

Comment:
Let format the imports. Please move thanos to the group below.

Reply:
Fixed.